### ISSUE TITLE
I want to know the initial learning rate.

### ISSUE THREAD

**Issue by liqiangseng** [if-loops/selective-synaptic-dampening#4]
Hi,could you tell me what the initial learning rate was when you pretrained the ViT model on CIFAR100? The initial learning rate has a significant impact on the accuracy, and this is very important for me. Thank you.
